todo_app/data/trello_items.py

```diff
@@ -0,0 +1,33 @@
+import os
+import requests
+from todo_app.item import Item
+
+board_url = "https://api.trello.com/1/boards/6212689e46f57218f07af552/lists/open"
```

Comment:
This `board_url` is specific to your board, which means that while your app looks customisable (because of the TRELLO_BOARD_KEY environment variables etc.) it actually only works for your Trello account. It would be nice to therefore make this board ID configurable as well